scripts/delete_site_data.py

```diff
@@ -0,0 +1,126 @@
+import argparse
```

Comment:
This is a copy of migration 5? Why not do a git-rename / delete of that file? (same question for the other script)

Reply:
Maybe a comment at the top of those files then? If a rando like me comes by in a year, I'll see two very similar files and be unclear on the history/reason. Though I suppose I can go to git log for that.

Reply:
ok, done.